### DISCUSSION TITLE
Docker Image with IMGT library

### DISCUSSION THREAD

**Discussion by val-axthelm** [milaboratory/mixcr#1651] [Q&A] [answered]
The latest docker image with imgt appears to be https://github.com/milaboratory/mixcr/pkgs/container/mixcr%2Fmixcr/28040254?tag=4.0.0-imgt. so 4.0.0. Is there plans to bundle newer image with imgt? Or instructions on how I can do so?

**Comment by mizraelson** [comment, marked as answer]
Hi,
You can download the latest imgt library from [here](https://github.com/repseqio/library-imgt/releases).
Then just put the file into the results directory that you mount to the `/work` one inside the docker.

In the example bellow I have the imgt library file `imgt.202312-3.sv8.json.gz` in `/Users/mizraelson/result`.

```
docker run --rm \
-e MI_LICENSE="XXXXXXXXXXXXXXXXXX" \
-v /Users/mizraelson/raw:/raw:ro \
-v /Users/mizraelson/result:/work \
ghcr.io/milaboratory/mixcr/mixcr:latest \
mixcr analyze neb-human-rna-xcr-umi-nebnext \
--library imgt \
/raw/Sample_{{R}}.fastq.gz \
Sample
```
